server/controllers/studentController.js

```diff
@@ -81,5 +94,48 @@ const updateStudent = async (req, res) => {
 
 }
 
+//generating token function
+const createToken = (_id) => {
+
+    return jwt.sign({ _id }, process.env.SECRET, { expiresIn: '3d' })
```

Comment:
Since you need the same timing for this login process, I would recommend even more to put this in an env or confg file